elements/rh-video/rh-video.css

```diff
@@ -0,0 +1,100 @@
+:host {
+  display: block;
+}
+
+:host([hidden]),
+*[hidden] {
+  display: none !important;
+}
+
+#video,
+#consent {
+  display: inline-flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+#video {
+  position: relative;
+  justify-content: stretch;
+}
+
+figure {
+  display: inline-flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+#consent,
+#watermark {
+  position: absolute;
+  inset: 0;
+}
+
+#consent {
+  justify-content: center;
+}
+
+#consent-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#consent-message,
+::slotted([slot="consent-message"]) {
+  font-family: var(--rh-font-family-heading, RedHatDisplay, "Red Hat Display", "Noto Sans Arabic", "Noto Sans Hebrew", "Noto Sans JP", "Noto Sans KR", "Noto Sans Malayalam", "Noto Sans SC", "Noto Sans TC", "Noto Sans Thai", Helvetica, Arial, sans-serif);
+  font-size: var(--rh-font-size-heading-sm, 1.5rem);
+  line-height: var(--rh-line-height-heading, 1.3);
+}
+
+figure,
+#consent-message,
+::slotted([slot="consent-message"]:last-of-type) {
+  margin-bottom: var(--rh-space-xl, 24px);
+}
+
+#play {
+  position: absolute;
+  inset: 0;
+  display: block;
+  padding: 0;
+  border: 0;
+  background-color: transparent;
+}
+
+#play[hidden],
+:is(.video, .consent) ::slotted([slot="thumbnail"]) {
+  opacity: 0;
+  pointer-events: none;
+}
+
+.visually-hidden {
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+#icon {
+  position: absolute;
+  inset: calc(50% - 32px);
+}
+
+#autoplay {
+  position: absolute;
+  inset: 0;
+  opacity: 0;
+  transition: opacity 200ms ease-in;
+}
+
+.video #autoplay {
+  opacity: 1;
+}
+
+::slotted([slot="caption"]:first-of-type) {
+  margin-top: var(--rh-space-xl, 24px);
```

Comment:
Caption text color should be `--rh-color-text-secondary-on-light` in light theme and `--rh-color-text-secondary-on-dark` in dark theme. Font size should be `var(--rh-font-size-body-text-sm, 0.875rem)`.

Caption's `margin-top` should be `var(--rh-space-lg, 16px)`.

Reply:
Addressed in [b5fa350](https://github.com/RedHat-UX/red-hat-design-system/pull/1514/commits/b5fa35014bef4f96db14b71f772652b9ff2a81a6).